src/snowflake/snowpark/functions.py

```diff
@@ -3636,7 +3636,8 @@ def date_sub(col: ColumnOrName, num_of_days: Union[ColumnOrName, int]):
 
 
 def datediff(part: str, col1: ColumnOrName, col2: ColumnOrName) -> Column:
-    """Calculates the difference between two date, time, or timestamp columns based on the date or time part requested.
+    """Calculates the difference between two date, time, or timestamp columns based on the date or time part requested, and
+    returns result of ``col2 - col1`` based on the requested date or time part.
```

Comment:
nit: should be single quote?
```suggestion
    returns result of `col2 - col1` based on the requested date or time part.
```

Reply:
In this sphinx doc, we need double back ticks to make font monospaced otherwise, it only gets italicized